utils/appliance/__init__.py

```diff
@@ -366,6 +366,8 @@ def _configure_5_3(self, log_callback=None):
         self.wait_for_web_ui(timeout=1800, log_callback=log_callback)
         self.loosen_pgssl(log_callback=log_callback)
         # self.ipapp.update_rhel(log_callback=log_callback)
+        if self.version >= '5.8':
```

Comment:
Until Pete merges that PR I can't really changes what I have because there is no call to configure_upstream (unless I'm missing something).